custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -74,6 +76,11 @@ def __init__(self, runtime):
         self.runtime = runtime
         self.options = runtime.options
         self.options.model_config = read_model_metadata_yaml(self.options.code_dir)
+        self.options.default_parameter_values = (
+            get_default_parameter_values(self.options.model_config)
```

Comment:
you might just want to handle the if else block in the function, also, you don't really need to set default_parameter_values here, you can just call the function when you need it below.

Reply:
I would prefer saving the default param values as the attributes of self.options and make it reusable throughout the life cycle of runner. We may handle the if/else within the function. But this will allow `model_metadata` as the only argument of the function `get_default_parameter_values` be None (So... Not sure if this is a good design of function)